src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+
+    public Square(Colors color, double side) {
+        super(color, side);
```

Comment:
It seems like the `super` constructor expects more than two parameters, or the parameters do not match the expected types. Ensure that the `Figure` class constructor is correctly defined to accept the parameters being passed here.